app/components/language-logo.hbs

```diff
@@ -1,4 +1,6 @@
-{{#if (eq @variant "color")}}
+{{#if (and (eq @language.slug "gleam") (eq @variant "color"))}}
+  <GleamLogo ...attributes />
```

Comment:
@Arpan-206 let's not assume that every color language logo usage of Gleam is going to be animated for now - let's keep this explicit (use `GleamLogo` instead of `LanguageLogo` in the tracks template). 

You can imagine using a color logo in things like user profile pages where we might not want this animation to occur

Reply:
@Arpan-206 this hasn't been addressed